api/httpapi/handlers/asset_handler_test.go

```diff
@@ -676,37 +676,60 @@ func TestAssetHandlerGet(t *testing.T) {
 			Description:  `should return http 500 if fetching fails`,
 			ExpectStatus: http.StatusInternalServerError,
 			Setup: func(ctx context.Context, ar *mocks.AssetRepository) {
-				ar.On("GetAll", ctx, asset.Config{}).Return([]asset.Asset{}, errors.New("unknown error"))
+				ar.On("GetAll", ctx, asset.Config{
+					Data: make(map[string]string),
```

Comment:
instead of doing this, is it possible to change the implementation to not instantiate a map if not needed?